server/plugin/plugin.go

```diff
@@ -969,26 +968,34 @@ func (p *Plugin) HasUnreads(info *GitHubUserInfo) bool {
 func (p *Plugin) checkOrg(org string) error {
 	config := p.getConfiguration()
 
-	configOrg := strings.TrimSpace(config.GitHubOrg)
-	if configOrg != "" && configOrg != org && strings.ToLower(configOrg) != org {
```

Comment:
We've changed the logic a bit here, because we set `configOrg` to lowercase in `getOrganizations`, which makes it so we can't do the case-sensitive compare here that was happening in the second part of the conditional i.e. `configOrg != org`. I'm thinking we should set `org` to lowercase as well when comparing. There may be other comparison things happening that need to take this into account as well